### PR TITLE
[Openshift] fixes addon sample pipelines

### PR DESCRIPTION
**Pull request by sm43**
This fixes the sample pipeline provided as part of addon.

Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: cmd/openshift/operator/kodata/tekton-addon/1.5.0/optional/samples/pipelinesamples/simple_pipeline.yaml [hunk 1]
```diff
@@ -8,34 +8,65 @@ spec:
...
+          runAfter:
+            - build
           params:
-            - name: PARAM
-              value: $(params.PARAM_NAME)
+            - name: ARGS
+              value:
+                - "new-app"
+                - "--docker-image"
+                - "$(params.IMAGE_NAME)"
```
Comment (nikhil-thomas) [inline cmd/openshift/operator/kodata/tekton-addon/1.5.0/optional/samples/pipelinesamples/simple_pipeline.yaml:72]: we are going to remove `ARGS` param and keep only `SCRIPT`  param.
could you handle this commad as a `SCRIPT`

Comment (sm43) [reply cmd/openshift/operator/kodata/tekton-addon/1.5.0/optional/samples/pipelinesamples/simple_pipeline.yaml]: updated.

**Review by vdemeester — APPROVED** [review]

**Pull request merged**